### PR TITLE
Rescope class names

### PR DESCRIPTION
**Pull request by epwinchell**
Jira: https://issues.redhat.com/browse/RHCLOUD-16591

- Add "hub" prefix to classes
- Use PF/Platform class naming convention

### CODE REVIEW

**Comment by epwinchell** [comment]
@himdel Please review/test. (I was not able to get an env working with adequate data for screenshots)

File: src/components/execution-environment-header/execution-environment-header.tsx [hunk 1]
```diff
@@ -42,7 +42,7 @@ export class ExecutionEnvironmentHeader extends React.Component<IProps> {
         pageControls={this.props.pageControls}
       >
         {last_sync_task && (
-          <p className='truncated'>
+          <p className='hub-m-truncated'>
```
Comment (himdel) [inline src/components/execution-environment-header/execution-environment-header.tsx:45]: there's also one `truncated` one line 54

**Comment by himdel** [comment]
Apart from that one missing `truncated`, and the merge commits, LGTM :+1: :)

**Comment by epwinchell** [comment]
PR replaced by: https://github.com/ansible/ansible-hub-ui/pull/1382

**Pull request closed without merging**